### DISCUSSION TITLE
Does payload has health check endpoint?

### DISCUSSION THREAD

**Discussion by gedaaaa** [payloadcms/payload#646] [General]
# Documentation Issue

Does payload has http health check endpoint? If we have, what's the path?

After some read of documentations, I didn't found related content. Health check would be very useful to work with AWS ECS and  ALB.

**Comment by denolfe** [comment]
Hey @gedaaaa, we see this as something that Payload shouldn't provide. However, because you have access to your own express app this can easily be added outside of Payload. Adding the following code to your app should work for you.

```ts
app.get('/health', (_, res) => {
  res.sendStatus(200);
});
```

Is this what you're looking for?

**Reply by gedaaaa** [reply]
Thanks!
